cnv/cnv_users_guide/cnv-dashboard.adoc

```diff
@@ -0,0 +1,13 @@
+[id="cnv-dashboard"]
+= Using the CNV Dashboard to get cluster information
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-dashboard
+toc::[]
+
+The Web Console is part of the {CNVProductName} interface.
+
+Access the {CNVProductName} Overview Dashboard, which contains high-level information about the cluster, by clicking *Home > Dashboards > Overview* from the {product-name} Web Console.
+
+The Overview dashboard provides a single, high-level view of cluster information, captured in individual dashboard sections or "cards".
```

Comment:
I think that the period should go inside the quotation marks here, since it's not code.

Reply:
we should actually underscore cards since we're introducing a new term:

    or _cards_.

Reply:
"dashboard" was previously capitalised.
I would drop "sections or" since they are only referred to as "cards", and that term is introduced in markup.

Reply:
Good calls. Thanks all. Done.